src/annotator/components/NotebookModal.tsx

```diff
@@ -112,10 +113,14 @@ export default function NotebookModal({
       closed={isHidden}
       onClose={onClose}
       data-testid="notebook-outer"
+      aria-labelledby={labelId}
     >
+      <div className="sr-only" id={labelId}>
+        Notebook dialog
+      </div>
```

Comment:
Considering this is only visible to screen readers, I suppose we could have used `aria-label="Notebook dialog"` instead, but since most of the docs suggest using `aria-labelledby`, I have decided to go with that.

Reply:
It might however make sense to add "Hypothesis" before Notebook / Profile, to make it clear that the dialog is part of Hypothesis and not the host page.